docs/hadoop-catalog.md

```diff
@@ -25,19 +25,88 @@ Hadoop 3. If there's any compatibility issue, please create an [issue](https://g
+| `s3-endpoint`                  | The endpoint of the AWS s3.                                                                                                                                                           | (none)          | Yes if it's a s3 fileset.  | 0.7.0-incubating |
+| `s3-access-key-id`             | The access key of the AWS s3.                                                                                                                                                         | (none)          | Yes if it's a s3 fileset.  | 0.7.0-incubating |
+| `s3-secret-access-key`         | The secret key of the AWS s3.                                                                                                                                                         | (none)          | Yes if it's a s3 fileset.  | 0.7.0-incubating |
+
+At the same time, you need to place the corresponding bundle jar [gravitno-aws-bundle-{version}.jar](https://repo1.maven.org/maven2/org/apache/gravitino/aws-bundle/) in the Hadoop environment.
+
+
+#### GCS fileset
+
+| Configuration item            | Description                                                                                                                                                                                                                  | Default value   | Required                   | Since version    |
+|-------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------|----------------------------|------------------|
+| `filesystem-providers`        | The file system providers to add. Set it to `gs` if it's a gcs fileset or a comma separated string that contains `gs` like `gs,s3` to support multiple kind of fileset including `gs`.                                       | (none)          | Yes                        | 0.7.0-incubating |
+| `default-filesystem-provider` | The name default filesystem providers of this Hadoop catalog if users do not specify the scheme in the URI. Default value is `builtin-local`, for GCS, if we set this value, we can omit the prefix 'gs://' in the location. | `builtin-local` | No                         | 0.7.0-incubating |
+| `gcs-service-account-file`    | The path of GCS service account JSON file.                                                                                                                                                                                   | (none)          | Yes if it's a gcs fileset. | 0.7.0-incubating |
+
+In the meantime, you need to place the corresponding bundle jar [gravitno-gcp-bundle-{version}.jar](https://repo1.maven.org/maven2/org/apache/gravitino/gcp-bundle/) in the Hadoop environment.
```

Comment:
You'd better specifically tell user where to put the jar, not vaguely saying "in the Hadoop environment".